### DISCUSSION TITLE
WinUEFI v2: Ideas

### DISCUSSION THREAD

**Discussion by Minionguyjpro** [Minionguyjpro/Easy-UEFI#10] [Ideas]
# WinUEFI v2: Ideas
Hey there! Soon we want to publish the new, big version of WinUEFI! But, we don't know what to change or where to start, so:
- Reply to this disscusion with an idea!
## Steps to post idea
1. Think about the idea. Is it or will it be good or possible?
2. Post the idea in your own words if you know.
3. We will check your idea. If it is good, we may pass it!

**Comment by juanpide** [comment]
Please, add an option to warn and confirm rebooting instead of just rebooting.
